Documentation/event-reference.md

```diff
@@ -0,0 +1,204 @@
+  - [Edge identity reset complete](#edge-identity-reset-complete)
+- [Events dispatched by Edge](#events-dispatched-by-edge)
+  - [Edge identity response](#edge-identity-response)
+  - [Edge content response error](#edge-content-response-error)
+  - [Edge event response](#edge-event-response)
+
+## Events handled by Edge
+
+The following events are handled by the Edge extension client-side.
+
+### Edge request content
+
+This event is a request to process and deliver an Experience event to Edge Network. This event is created when the `Edge.sendEvent(ExperienceEvent)` API is called. This event is captured by the Edge Network extension's event listener in the Event Hub for processing and sent to the Edge Network.
+
+If the required `xdm` key is not present in the event data payload, the event is not sent to Edge Network.
+
+#### Event Details<!-- omit in toc -->
+
+| API      | Event type         | Event source      |
+| -------- | ------------------ | ----------------- |
+| [`Edge.sendEvent(experienceEvent,callback)`](api-reference.md#sendevent) | com.adobe.eventType.edge | com.adobe.eventSource.requestContent |
+
+#### Data payload definition<!-- omit in toc -->
+
+| Key       | Value type    | Required | Description           |
+| --------- | ------------- | -------- | --------------------- |
+| xdm       | `Map<String, Object>` | Yes      | XDM formatted data; use an `XDMSchema` implementation for a better XDM data ingestion and format control. |
+| data      | `Map<String, Object>` | No       | Optional free-form data associated with this event. |
+| datasetId | `String`        | No       | Optional custom dataset ID. If not set, the event uses the default Experience dataset ID set in the datastream configuration |
+
+> **Note**  
+> Events of this type and source are only processed if the data collection consent status stored in the `collect` property is **not** `n` (no); that is, either yes (`y`) or pending (`p`).
+
+-----
+
+### Edge request identity
+This event is a request to get the current location hint being used by the Edge Network extension in requests to the Edge Network. The Edge Network location hint may be used when building the URL for Edge Network requests to hint at the server cluster to use.
+
+#### Event Details<!-- omit in toc -->
+
+| API      | Event type         | Event source      |
+| -------- | ------------------ | ----------------- |
+| [`Edge.getLocationHint(callback)`](api-reference.md#getlocationhint) | com.adobe.eventType.edge | com.adobe.eventSource.requestIdentity |
+
+#### Data payload definition<!-- omit in toc -->
+
+| Key          | Value type    | Required | Description                |
+| ------------ | ------------- | -------- | -------------------------- |
+| locationHint | `boolean`     | Yes      | The Edge Network location hint to use when connecting to Edge Network. Property is set to `true` automatically; it is not user modifiable. |
```

Comment:
This field is used to distinguish (edge, requestIdentity) event  is a getter for locationHint. Let's update the description accordingly.

Reply:
We can add Required value as `Yes (internal)`

Reply:
Makes sense. We can keep it as is then.